src/System.Text.Json/ref/System.Text.Json.csproj

```diff
@@ -6,8 +6,11 @@
   <ItemGroup>
     <Compile Include="System.Text.Json.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
     <ProjectReference Include="..\..\System.Memory\ref\System.Memory.csproj" />
     <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
+    <Reference Include="System.Memory" />
```

Comment:
This should be a ProjectReference since it is part of the ref build.

Reply:
That fails to build:
```text
C:\Users\ahkha\.nuget\packages\microsoft.dotnet.build.tasks.configuration\1.0.0-beta.19064.4\build\Microsoft.DotNet.Build.Tasks.Configuration.targets(166,5): error : Could not find a configuration for ProjectReference '..\..\System.Memory\ref\System.Memory.csproj' from configurations 'uap;netcoreapp' when building 'System.Text.Json' for configuration 'netstandard' . [E:\GitHub\Fork\corefx\src\System.Text.Json\ref\System.Text.Json.csproj]
```

Reply:
I don't to see any build failures (locally or in CI) with the PR as is, so I am not sure if such a change is necessary.

Reply:
I guess just in this case, this is fine because System.Memory is actually comming from the netstandard targeting pack, so that should be fine. My primary concern was that in refs, you should always try to use ProjectReference in order to avoid a race condition.